docs/proposals/old-timer.md

```diff
@@ -0,0 +1,212 @@
+# Heapster Oldtimer
+
+## Overview
+
+Prior to the Heapster refactor, the Heapster model presented aggregations of
+metrics over certain time periods (the last hour and day).  Post-refactor, the
+concern of presenting an interface for historical metrics was to be split into
+a separate Heapster component: Oldtimer.
+
+Oldtimer will run as part of the main Heapster executable, and will present
```

Comment:
Oldtimer should rather run as a separate executable/containert in the main Heapster pod.